telemetry/telemetry_test.go

```diff
@@ -75,7 +75,10 @@ func TestPreview(t *testing.T) {
 	jsonParsed, err := gabs.ParseJSON([]byte(r))
 	require.NoError(t, err)
 	require.Equal(t, trackingID, jsonParsed.Path("trackingId").Data().(string))
-	require.True(t, jsonParsed.ExistsP("hostExtra.cpuFlags"))
+	// Apple M1 doesn't contain cpuFlags
+	if runtime.GOARCH != "arm" && runtime.GOOS != "darwin" {
```

Comment:
@hawkingrei @xiongjiwei @bb7133 is this condition correct? I think it skip the test even if on Apple Intel or any other ARM arch os.

Reply:
fixed in https://github.com/pingcap/tidb/pull/26826